nilearn/datasets/func.py

```diff
@@ -907,107 +903,79 @@ def fetch_localizer_contrasts(contrasts, n_subjects=None, get_tmaps=False,
-        for data_type in data_types:
-            for contrast_id, contrast in enumerate(contrasts_wrapped):
-                name_aux = str.replace(
-                    str.join('_', [data_type, contrast]), ' ', '_')
-                file_path = os.path.join(
-                    "brainomics_data", subject_id, "%s.nii.gz" % name_aux)
-                file_tarball_url = urls[contrast_id]
-                filenames.append((file_path, file_tarball_url, opts))
-    # Fetch masks if asked by user
-    if get_masks:
-        urls.append("%sbrainomics_data_masks.zip?rql=%s&vid=data-zip"
-                    % (root_url,
-                       _urllib.parse.quote(base_query % {"types": '"boolean mask"',
-                                                  "label": "mask"},
-                                    safe=',()')))
-        for subject_id in subject_ids:
-            file_path = os.path.join(
-                "brainomics_data", subject_id, "boolean_mask_mask.nii.gz")
-            file_tarball_url = urls[-1]
-            filenames.append((file_path, file_tarball_url, opts))
-    # Fetch anats if asked by user
-    if get_anats:
-        urls.append("%sbrainomics_data_anats.zip?rql=%s&vid=data-zip"
-                    % (root_url,
-                       _urllib.parse.quote(base_query % {"types": '"normalized T1"',
-                                                  "label": "anatomy"},
-                                    safe=',()')))
-        for subject_id in subject_ids:
-            file_path = os.path.join(
-                "brainomics_data", subject_id,
-                "normalized_T1_anat_defaced.nii.gz")
-            file_tarball_url = urls[-1]
-            filenames.append((file_path, file_tarball_url, opts))
-    # Fetch subject characteristics (separated in two files)
-    if url is None:
-        url_csv = ("%sdataset/cubicwebexport.csv?rql=%s&vid=csvexport"
-                   % (root_url, _urllib.parse.quote("Any X WHERE X is Subject")))
-        url_csv2 = ("%sdataset/cubicwebexport2.csv?rql=%s&vid=csvexport"
-                    % (root_url,
-                       _urllib.parse.quote("Any X,XI,XD WHERE X is QuestionnaireRun, "
-                                    "X identifier XI, X datetime "
-                                    "XD", safe=',')
-                       ))
-    else:
-        url_csv = "%s/cubicwebexport.csv" % url
-        url_csv2 = "%s/cubicwebexport2.csv" % url
-    filenames += [("cubicwebexport.csv", url_csv, {}),
-                  ("cubicwebexport2.csv", url_csv2, {})]
 
-    # Actual data fetching
-    dataset_name = 'brainomics_localizer'
+    # Define the download url and perform the download operation
+    # We use the TXT extraction mode to get a list of files that can be
+    # downloaded. We use this option in order to be able to filter
+    # localy the dataset.
+    # TODO: use the 'zone partenaire' server root url
+    root_url = url or "http://localhost:9090/explore?project=localizer"
```

Comment:
This URL look wrong, doesn't it?

Reply:
Yes.
I configured apache on my local machine so that it is possible to access the data if you want to perform some tests.